haiku-driver.groovy

```diff
@@ -92,25 +164,27 @@ def setSpeed(fanspeed){
             sendFanSpeedCommand(7)
             break
     }
-
-    sendEvent(name: "speed", value: "${fanspeed}", isStateChange: true)
 }
 
 def sendFanPowerCommand(String command) {
     sendCommand("FAN", "PWR", command)
 }
 
+def refreshFanSpeed() {
+    sendCommand("FAN", "SPD", "GET;ACTUAL")
```

Comment:
Why is this separate from `refresh` above?

Reply:
Because I also refresh the fan speed whenever the fan is turned on, since the fan itself will remember its previous speed.

Reply:
So since this is called from LIGHT PWR and FAN DIR, this event should be triggered if those events occur?

Reply:
Nice catch. Should be in FAN PWR and FAN DIR. derp.